### PR TITLE
Welcome page

### PR DESCRIPTION
**Pull request by kathytbui**
### New Feature Submissions:
* [x] Have you added an explanation of what your changes do?
* [x] Have you linted your code locally prior to submission?

### Testing:
* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?
* [x] All new and existing tests passed?

**Test coverage**
- 2 tests added - 0 failures - all passing
- simplecov - 100% test coverage
- rubocop - no offenses
--------------------------------------------------------------------------------
### Change Log:

- Created welcome spec to test what a visitor would see
- Spec includes navbar features tests
- Spec includes testing for welcome page content including:
  - Description of website
  - Website name header
  - Placeholder image of our staple image
- Add sandbox folder to .gitignore
- Add additional travisCI configurations to .travis.yml
- Added a navbar to the application.html.erb
- Updated Welcome index to include description, image, and login button

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,5 +7,8 @@ script:
 - bundle exec rails db:{create,migrate} RAILS_ENV=test
 - bundle exec rspec
 deploy:
+  provider: heroku
```
Comment (dtsehl) [inline .travis.yml:10]: Good looking out to add this in!

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -10,6 +10,22 @@
...
+          </li>
+        </ul>
+        <ul class="navbar-login-links">
+          <li class="nav">
+            <a class="nav-link" href="/auth/google_oauth2">Login</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+
```
Comment (sciencefixion) [inline app/views/layouts/application.html.erb:28]: Great start with all this nav! :)

**Review by janegreene — APPROVED** [review]
Looks good, nice work!

**Review by sciencefixion — COMMENTED** [review]
Great work! We're off!

**Pull request merged**